src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -1001,6 +1015,23 @@ export default {
 				this.getTributeContainer().classList.remove(this.$style['tribute-container--focus-visible'])
 			}
 		},
+
+		showTributeManually(trigger) {
```

Comment:
If this is a public method, please, add documentation in JSDoc with a description and `@public` tag

Reply:
Same for hide